src/CssInliner.php

```diff
@@ -616,6 +619,9 @@ private function getNodesToExclude(): array
                 continue;
             }
             foreach ($matchingNodes as $node) {
+                if (!$node instanceof \DOMElement) {
+                    throw new \UnexpectedValueException('$node is on DOMElement.', 1617975914);
```

Comment:
There's a typo here.  Should be "is not a DOMElement"?  Also, we could give more info in the message by using `$node->getNodePath() ?? '$node'`.

Reply:
Done.